qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -94,3 +94,19 @@
         - used_netdev:
             used_sameid = yes
             devadd_match_string = "it's in use"
+        - with_iommu:
+            only virtio_net
+            only nic_virtio
+            netdev_extra_params_hotplug_nic1 = ",vhost=on,vhostforce=on"
```

Comment:
`vhost` and `vhostforce` can be standalone params.

Reply:
Do you mean to write like the following? However, writing the latter value will overwrite the previous one.
netdev_extra_params_hotplug_nic1 = ",vhost=on"
netdev_extra_params_hotplug_nic1 = ",vhostforce=on"

Reply:
Oh, I mean `vhost_nic1 = on` and `vhostforce_nic1 = on`

Reply:
Hi @PaulYuuu, The hotplug device needs to use the existing usage. Otherwise it will not appear in the hotplug command line。

Reply:
Okay, this should be an enhancement on `qemu_vm.hotplug_nic()`, can skip this enhancement this time.